app/public/src/pages/game.tsx

```diff
@@ -102,6 +105,45 @@ export function getGameScene(): GameScene | undefined {
     | undefined
 }
 
+export function cyclePlayers(amt: number) {
+  logger.info(`cycling players by ${amt}`)
+  const players = values(gameContainer.room?.state.players)
+  logger.info(values(gameContainer.room?.state.players))
+  let nextPlayer = players.findIndex((p) => p === gameContainer.player) + amt
+  logger.info(nextPlayer)
+  while (nextPlayer >= players.length) {
```

Comment:
trick to avoid these double while loops:

```suggestion
let nextPlayer = (players.findIndex((p) => p === gameContainer.player) + amt + players.length) % players.length
```